### DISCUSSION TITLE
nitropack as devDependencies or dependencies

### DISCUSSION THREAD

**Discussion by oumarbarry** [nitrojs/nitro#1786] [Q&A] [answered]
Maybe it's not that important, but to clarify things once for all,

- by using for example, the starter template: `bunx giget@latest nitro nitro-app`, we have:
```json
"dependencies": {
    "nitropack": "latest"
}
```

- but for example, in the [edge release channel section of the docs](https://nitro.unjs.io/guide/getting-started#edge-release-channel), we have this:
```json
{
  "devDependencies": {
--    "nitropack": "^2.0.0"
++    "nitropack": "npm:nitropack-edge@latest"
  }
}
```

**Comment by pi0** [comment, marked as answer]
Hi. In the end, it doesn't matter because you have to deploy `.output` directory only.

It is in `devDependencies` as recommended because if by accident (or requirement of some deployment providers) you really have to deploy the entire project, `nitropack` wouldn't be installed in production runtime but you might also choose to use `dependencies` to split between development tools and frameworks.
